tests/rkt_selinux_test.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2016 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"github.com/coreos/rkt/pkg/selinux"
+	"github.com/coreos/rkt/tests/testutils"
+)
+
+func TestSelinuxMount(t *testing.T) {
+	osInfo := getOSReleaseOrFail(t)
+	if osInfo["ID"] != "fedora" || osInfo["VERSION_ID"] != "25" {
```

Comment:
This will also work on CoreOS (probably).

But we don't have an easy way to run functional tests there so you can ignore this comment for now :)